codegen/sdk-codegen/sdk-partitions.json

```diff
@@ -9,7 +9,7 @@
       "supportsDualStack" : true,
       "supportsFIPS" : true
     },
-    "regionRegex" : "^(us|eu|ap|sa|ca|me|af|il)\\-\\w+\\-\\d+$",
+    "regionRegex" : "^(us|eu|ap|sa|ca|me|af|il|mx)\\-\\w+\\-\\d+$",
```

Comment:
This file was already present, but has apparently not been modified for some time.
This mod will keep this copy of the partitions file up-to-date.

Reply:
The `Partitions.swift` file is created using the contents of this file.